src/networks.json

```diff
@@ -950,6 +965,19 @@
     "explorer": "https://snowtrace.io",
     "start": 536483,
     "imageIPFS": "QmeS75uS7XLR8o8uUzhLRVYPX9vMFf4DXgKxQeCzyy7vM2"
+  },
+   "47805": {
+    "key": "47805",
+    "name": "REI Mainnet",
+    "chainId": 47805,
+    "network": "mainnet",
+    "multicall": "0x9eE9904815B80C39C1a27294E69a8626EAa7952d",
+    "rpc": [
+      "https://rpc.rei.network"
```

Comment:
Hi @bijianing97 Is there CROS origin enabled on this node, i get this error:
```
Error: could not detect network (event="noNetwork", code=NETWORK_ERROR, version=providers/5.4.5)
```

Reply:
Sorry to that, now the CROS origin is worked, I will be very grateful if you try it again